.github/ISSUE_TEMPLATE/story.md

```diff
@@ -29,4 +29,5 @@ Add any other context or screenshots about the story here.
 
 ## Definition of Done
 
-- [ ] _TBD._
+- [x] Engineering
```

Comment:
We should actually have our definition of done list in here.  E.g.
```markdown
- [ ] Threat model updated
- [ ] Code refactored for clarity and no design/technical debt
- [ ] Adhere to separation of concerns; code is not tightly coupled, especially to 3rd party dependencies.
```
Etcetera.

Reply:
I like that idea, good point.